### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by carusogabriel**
Together with PHPStorm and https://github.com/kalessil/phpinspectionsea, here are some improvements that we can make to PHP_CodeSniffer :smile:

### CODE REVIEW

File: src/Files/File.php [hunk 1]
```diff
@@ -776,11 +776,7 @@ public function addFixableError(
         $severity=0
     ) {
         $recorded = $this->addError($error, $stackPtr, $code, $data, $severity, true);
-        if ($recorded === true && $this->fixer->enabled === true) {
-            return true;
-        }
-
-        return false;
+        return $recorded === true && $this->fixer->enabled === true;
```
Comment (gsherwood) [inline src/Files/File.php]: This isn't how I write code. I prefer to use the IF statement for my own readability.

**Comment by gsherwood** [comment]
If you want to adjust the PR, I'd merge in stripos change, the fixed var name, and the unused imports. But I write my return statements like they are very munch on purpose, so I wouldn't merge those.

**Comment by carusogabriel** [comment]
@gsherwood Done :+1:

**Comment by gsherwood** [comment]
Thanks for doing that for me.

**Pull request merged**